src/test/scala/mesosphere/marathon/raml/PodStatusConversionTest.scala

```diff
@@ -17,6 +17,19 @@ class PodStatusConversionTest extends UnitTest {
 
   import PodStatusConversionTest._
 
+  "PodConversion" should {
+    "keep linux info on executor" in {
+      val pod = basicOneContainerPod.copy(linuxInfo = Some(state.LinuxInfo(seccomp = None, ipcInfo = Some(state.IPCInfo(ipcMode = state.IpcMode.Private, shmSize = Some(32))))))
+
+      val ramlPod = pod.toRaml
+
+      val ramlLinuxInfo = Some(LinuxInfo(seccomp = None, ipcInfo = Some(IPCInfo(mode = IPCMode.Private, shmSize = Some(32)))))
+
+      ramlPod.linuxInfo should be(ramlLinuxInfo)
```

Comment:
Do we have a fromRAML method? Perhaps, instead, we can do a round trip conversion, asserting that both conversions work properly.

Reply:
Added a roundtrip unit test